crap-code.kt

```diff
@@ -0,0 +1,26 @@
+package badcode.example
+class PerformanceFile {
+    val largeList = List(1000000) { it }
+    // Inefficient way to calculate the sum of elements
+    var sum = 0
+    fun calculateSum() {
+        for (i in 0 until largeList.size) {
+            sum += largeList[i]
+        }
+        // typo in println() function
+        prentln("Sum: $sum")
+        // Unnecessary object creation inside a loop
+        for (i in 0 until 100000) {
+            val tempObject = Any() // Creates a new object on each iteration
+        }
```

Comment:
Avoid creating unnecessary objects inside the loop to improve performance.
```suggestion
for (i in 0 until 100000) {
    // Perform operations without creating a new object
}
```